Calibration/IsolatedParticles/plugins/StudyCaloResponse.cc

```diff
@@ -328,7 +328,11 @@ void StudyCaloResponse::analyze(edm::Event const& iEvent, edm::EventSetup const&
   bool ok(false);
   std::string triggerUse("None");
   if (!triggerEventHandle.isValid()) {
-    edm::LogWarning("IsoTrack") << "Error! Can't get the product " << triggerEvent_.label();
+    if (trigNames_.empty()) {
+      ok = true;
+    } else {
+      edm::LogWarning("IsoTrack") << "Error! Can't get the product " << triggerEvent_.label();
```

Comment:
```suggestion
      edm::LogWarning("StudyCaloResponse") << "Error! Can't get the product " << triggerEvent_.label();
```

Sorry, I just noticed this, the whole file is using `edm::LogWarning("IsoTrack")` but the plugin name is  `StudyCaloResponse`, so it should be that passed to the `LogWarning`